pkg/object/meshcontroller/service/service.go

```diff
@@ -333,14 +334,14 @@ func (s *Service) DeleteServiceInstanceSpec(serviceName, instanceID string) {
 // ListTenantSpecs lists tenant specs
 func (s *Service) ListTenantSpecs() []*spec.Tenant {
 	tenants := []*spec.Tenant{}
-	kvs, err := s.store.GetPrefix(layout.TenantPrefix())
+	kvs, err := s.store.GetRawPrefix(layout.TenantPrefix())
```

Comment:
`GetRawPrefix` actually returns all Etcd metadata for one key, and `GetPrefix` casts them away by keeping only the value. So if u don't need those metadata, then I can't get why we need to replace it here.

Reply:
this is because, for example, `GetTenantSpecWithInfo` is using the `raw` APIs, I want to make them align.